dockerfiles/incr.sh

```diff
@@ -30,7 +30,10 @@ PREV=$1
 
 BUILD_TYPE=Debug
 BUILD_LABEL=${HEAD}-Debug
-GITHUB_USER=$(git remote get-url origin | sed s/.*:// | sed s+/.*++)
+GITHUB_USER=$(git remote get-url origin | sed -e "s/http.*com\\///" -e "s/.*://" -e s+/.*++)
```

Comment:
Minor, but I think you are using \\/ in your sed expression to indicate a /, where later in the same line I have used + in place of / as my delimiter in order to be able to include / in the search string

For consistency it would be better if you say "s+http.*com/++"